README.md

```diff
@@ -304,6 +304,47 @@ In view:
 
 <?php echo $form->field($model->getModel('profile'), 'middlename')->textInput(['maxlength' => 255]) ?>    
 ```
+
+### Queue
+
+Basic [Queueing](https://github.com/yiisoft/yii2-queue) component
+implementation a file based  queueing service.
+
+Additional module [readme](https://github.com/yiisoft/yii2-queue/blob/master/README.md).
+
+Implement Queue class
+
+```php
+class DownloadJob extends BaseObject implements \yii\queue\JobInterface
+{
+    public $url;
+    public $file;
+
+    public function execute($queue)
+    {
+        file_put_contents($this->file, file_get_contents($this->url));
+    }
+}
+```
+
+Here's how to send a task into queue:
+
+```php
+Yii::$app->queue->push(new DownloadJob([
+    'url' => 'http://example.com/image.jpg',
+    'file' => '/tmp/image.jpg',
+]));
+```
+
+Pushes job into queue that run after 5 min:
+
+```php
+Yii::$app->queue->delay(5 * 60)->push(new DownloadJob([
+    'url' => 'http://example.com/image.jpg',
+    'file' => '/tmp/image.jpg',
+]));
+```
+
```

Comment:
can you add a few words about how to start listeners? it doubtless that we'll have questions about it ;-)

Reply:
Added below.